packages/atst/docs/sdk.md

```diff
@@ -52,21 +52,26 @@ createClient({
 
 ### Reading an attestation
 
+To read an attestation use `readString`, `readAddress`, `readNumber`, `readBool`
+
+Use `readAttestation` to with `bytes` passed in for the types parameter to read raw bytes
```

Comment:
Nit: Remove "to". Also it says use readAttestation but in the example it uses readAttestationString. Can we update the sentence to make it clearer?

Reply:
```suggestion
Use `readAttestationString` with `bytes` passed in for the types parameter to read raw bytes
```
nice catch